klasses/conftest.py

```diff
@@ -16,3 +27,39 @@ def mocked_requests_get(mocker):
     mocked_response = Mock()
     mocked_get_func.return_value = mocked_response
     return SimpleNamespace(request=mocked_get_func, response=mocked_response)
+
+
+@pytest.fixture()
+def patched_create_run_enrollments(mocker):
+    """patched create_run_enrollments"""
+    return mocker.patch("klasses.api.create_run_enrollments", return_value=([], False))
+
+
+@pytest.fixture()
+def patched_deactivate_run_enrollment(mocker):
+    """patched deactivate_run_enrollment"""
+    return mocker.patch("klasses.api.deactivate_run_enrollment", return_value=[])
+
+
+@pytest.fixture()
+def enrollment_data(user):
```

Comment:
If any of these new fixtures are only used in `klasses/api_test.py`, we should define them in that module instead of here

Reply:
@gsidebo I have used it in the same file but it was generating that so decided to move it in `conftest`

```
___________________________________ [pylint] klasses/api_test.py ____________________________________
W:311, 4: Redefining name 'patched_create_run_enrollments' from outer scope (line 303) (redefined-outer-name)
W:332, 4: Redefining name 'patched_create_run_enrollments' from outer scope (line 303) (redefined-outer-name)
W:353, 4: Redefining name 'patched_create_run_enrollments' from outer scope (line 303) (redefined-outer-name)
W:374, 4: Redefining name 'patched_create_run_enrollments' from outer scope (line 303) (redefined-outer-name)
W:395, 4: Redefining name 'patched_create_run_enrollments' from outer scope (line 303) (redefined-outer-name)
W:416, 4: Redefining name 'patched_create_run_enrollments' from outer scope (line 303) (redefined-outer-name)```

Reply:
We typically ignore those linting errors (`redefined-outer-name`) in Python test files. You can grep the codebase for the many places where we ignore it. It has to do with the fundamental difference between the way that function arguments in Python are normally used and the way that they're used in pytest test cases.